### PR TITLE
cjxl, djxl: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/cjxl.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# cjxl
+
+> Compress images to JPEG XL.
+> More information: <https://github.com/libjxl/libjxl>.
+
+- Convert an image to JPEG XL:
+
+`cjxl {{path/to/image}} {{path/to/target.jxl}} `
```
Comment (spageektti) [inline pages/linux/cjxl.md:8]:
```suggestion
`cjxl {{path/to/image}} {{path/to/output.jxl}} `
```

File: pages/linux/cjxl.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Maximize the quality and compression of the resulting image:
+
+`cjxl --distance 0 --effort 9 {{path/to/image}} {{path/to/target.jxl}}`
```
Comment (spageektti) [inline pages/linux/cjxl.md:12]:
```suggestion
`cjxl --distance 0 --effort 9 {{path/to/image}} {{path/to/output.jxl}}`
```

**Review by spageektti — APPROVED** [review]
Hi. Thanks for your contribution.
I think using "target" for the output file is in this case confusing.

**Comment by Managor** [comment]
Still adding some examples as I get suggestions.

File: pages/linux/cjxl.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# cjxl
+
+> Compress images to JPEG XL.
+> More information: <https://github.com/libjxl/libjxl>.
+
+- Convert an image to JPEG XL:
+
+`cjxl {{path/to/image}} {{path/to/output.jxl}}`
+
+- Maximize the quality and compression of the resulting image:
```
Comment (kkthompson) [inline pages/linux/cjxl.md:10]: Losslessly maximize…?

**Comment by spageektti** [comment]
Do you know what type of file is accepted as an input image? If there are only some, we could specify that.

**Comment by kkthompson** [comment]
Inputs are: PNG, APNG, GIF, JPEG, EXR, PPM, PFM, PAM, PGX, or JXL

File: pages/linux/cjxl.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# cjxl
+
+> Compress images to JPEG XL.
```
Comment (spageektti) [inline pages/linux/cjxl.md:3]:
```suggestion
> Compress images to JPEG XL.
> Accepted input formats are PNG, APNG, GIF, JPEG, EXR, PPM, PFM, PAM, PGX, or JXL.
```

```suggestion
> Compress images to JPEG XL.
> Accepted input extensions are PNG, APNG, GIF, JPEG, EXR, PPM, PFM, PAM, PGX, or JXL.
```

**Review by spageektti — APPROVED** [review]

File: pages/linux/cjxl.md (hunk 3 above)
Comment (Managor) [reply pages/linux/cjxl.md]: I did a rewording that is probably better.

File: pages/linux/cjxl.md (hunk 4 above)
Comment (Managor) [reply pages/linux/cjxl.md:3]: "extensions" is probably better since we're using `.ext` in a command.

**Comment by Managor** [comment]
This command was uniform on all platforms right? If so, I should move this to common.

**Pull request merged**